core/cmd/presenters.go

```diff
@@ -0,0 +1,161 @@
+package cmd
```

Comment:
any reason we can just put this in the presenters package with the other ones? Especially since cmd already depends on it. I actually think the presenters package could benefit from some restructuring also - namely splitting the single presenters.go file into a file per specific presenter (this could be a job_v2.go file in there) to make it easier to find stuff.